.github/workflows/fuzz.yml

```diff
@@ -7,6 +7,9 @@ on:
   pull_request:
     branches:
       - main
+  workflow_dispatch:
+    branches:
```

Comment:
Why only on main? This makes the trigger a little pointless. The idea is that you can press the button on any branch, i.e. code that doesn't trigger the CI automatically.